content/05.dl-complexities.md

```diff
@@ -1 +1,25 @@
 ## Tip 3: Understand the complexities of training deep neural networks {#complexities}
+
+Correctly training deep neural networks is a non-trivial process.
+There are many different options and potential pitfalls at every stage.
+To get good results you have to expect to train many networks with
+different tweaks and modifications.
+Despite improved framework ease-of-use and on-demand cloud computing resources
+this means DL can be very demanding, requiring significant
+infrastructure and patitence to complete.
+The experimentation inherent to DL is often noisy (requiring repetition) and
+represent a significant organisational challenge.
+All code, parameters, and results must be carefully corralled using good
+coding practices (e.g. version control, continuous integration etc.) in order to 
+be effective and interpretable.
+This organisation is also key to being able to efficiently share your work
+and to update your model as new data becomes available. 
+
+
+Similar to [Tip 4](#baselines), try to start with a relatively smaller network
```

Comment:
Not sure if we were 100% consistent about that, but I think we referred to the reader in 3rd person before ("the researcher", "the experimenter," etc.)

Reply:
Might be easiest to fix that flow/match tone when we have a draft for all sections?

Reply:
I agree. Right now, these things are a bit difficult to address without having the context of all chapters!

Reply:
Sounds good. Like you suggested before, it may be better to wait until we have all the tips outlined in a first draft before we smooth it out to make it consistent :)